pkg/storage/etcd/testing/utils.go

```diff
@@ -164,8 +311,8 @@ func (m *EtcdTestServer) Terminate(t *testing.T) {
 }
 
 // NewEtcdTestClientServer creates a new client and server for testing
-func NewEtcdTestClientServer(t *testing.T) *EtcdTestServer {
-	server := configureTestCluster(t, "foo")
+func NewEtcdTestClientServer(t *testing.T, secure bool) *EtcdTestServer {
+	server := configureTestCluster(t, "foo", secure)
```

Comment:
So I debate if there is ever a case where we don't want to default to secure for *.  @lavalamp does google support non-sec deploys?  Red Hat does not.